src/test/basics/PerfLog_test.cpp

```diff
@@ -280,55 +280,44 @@ class PerfLog_test : public beast::unit_test::suite
             if (! BEAST_EXPECT(! ec))
                 return;
 
-            std::ofstream woFile;
-            woFile.open (fullPath.c_str(), std::ios::out | std::ios::app);
-            if (! BEAST_EXPECT(woFile))
+            auto fileWriteable = [](boost::filesystem::path const& p)
+            {
+                return std::ofstream {p.c_str(), std::ios::out | std::ios::app};
```

Comment:
This lambda returns a `ofstream`. We want it to return a bool.

Reply:
Good point.  I have a fix coming.